contributed_definitions/nyaml/NXactivity.yaml

```diff
@@ -0,0 +1,22 @@
+category: base
+doc: |
+  A planned or unplanned action that has a temporal extension and for some time depends on some entity.
```

Comment:
Add a line break in between the first and all following lines as the first line will be parsed out for the summary in the documentation always